src/pyOpenMS/dist-scripts/create-manylinux.sh

```diff
@@ -4,12 +4,26 @@
 #
 # Execute as:
 # 
-#   sudo docker run --net=host -v `pwd`:/data hroest/manylinux_qt_contrib /bin/bash /data/create-manylinux.sh
+#   sudo docker run --net=host -v `pwd`:/data hroest/manylinux_qt_contrib:v2 /bin/bash /data/create-manylinux.sh
 #
 
-git clone -b Release2.2.0 https://github.com/OpenMS/OpenMS.git
+set -e
+
+# Note that we cannot use git any more as it refuses to communicate with
+# github, but we have a patched version of wget with a newer OpenSSL version
+# capable of downloading the release tar file.
+wget https://github.com/OpenMS/OpenMS/releases/download/Release2.3.0/OpenMS-2.3.0-src.tar.gz -O OpenMS-2.3.0-src.tar.gz
```

Comment:
can you add to https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release that this file needs to be adapted

Reply:
done

@grosenberger can you comment on the OSX section here: https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release#pyopenms_release